script.js

```diff
@@ -0,0 +1,27 @@
+// one-var
+
+var a = 10;
+
+const value = 10;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['value' is assigned a value but never used.](https://app.codacy.com/gh/WebHeroSchool/js_test2/pullRequest?prid=5502592)